Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -659,5 +659,34 @@ function OreMine_GetSpriteName(furniture)
     return "mine_" .. furniture.Parameters["ore_type"].ToString()
 end
 
+-- This function gets called once, when the furniture is installed
+function Rtg_InstallAction( furniture, deltaTime)
+    -- TODO: find elegant way to register heat source and sinks to Temperature
+	furniture.EventActions.Register("OnUpdateTemperature", "Rtg_UpdateTemperature")
+	World.Current.temperature.RegisterSinkOrSource(furniture)
+end
+
+-- This function gets called once, when the furniture is uninstalled
+function Rtg_UninstallAction( furniture, deltaTime)
+    furniture.EventActions.Deregister("OnUpdateTemperature", "Rtg_UpdateTemperature")
+	World.Current.temperature.DeregisterSinkOrSource(furniture)
+	-- TODO: find elegant way to unregister previous register
+end
+
+function Rtg_UpdateTemperature( furniture, deltaTime)
```

Comment:
Okay... this is literally the same code as Heater_UpdateTemperature(furniture, deltaTime). So why don't you call that one instead, either from Rtg_UpdateTemperature or directly?